dev_docs/developer_scripts.md

```diff
@@ -118,3 +70,65 @@ gui-spec-windows - creates the initial spec file for windows, do not run unless
 gui-build-osx - builds the app from the osx spec file
 gui-build-windows - builds the app from the windows spec file
 ```
+## Client Makefile
+
+The following phony `make` targets in `client/Makefile` are convenience methods for getting you up and developing.
+
+### Runner scripts
+
+#### start-frontend
+
+**About** Serve the current client javascript independently from the `server` code.
+
+**Requires**
+* The server to be running. Best way to do this is with `make backend-dev`
+* `make ci` to install the necessary node modules
```

Comment:
`make ci` is a wipe and install of all required node modules, should it instead be `npm install`?

Reply:
The idea here is that you don't always want to update your `package-lock.json` when developing and definitely not while you're running CI. For more see https://docs.npmjs.com/cli/ci.html